tv_apps/browser/js/awesomescreen.js

```diff
@@ -1275,6 +1286,23 @@ var Awesomescreen = {
     Awesomescreen.showPointerImg();
   },
 
+  handleinputAreaInput: function awesomescreen_handleinputAreaInput(ev) {
+    if(Awesomescreen.isInputAreaValid()) {
+      Awesomescreen.bmtitleArea.classList.remove('invalid');
+      Awesomescreen.okButton.classList.remove('disable');
+      Awesomescreen.renameConfirmButton.classList.remove('disable');
+    } else {
+      Awesomescreen.bmtitleArea.classList.add('invalid');
+      Awesomescreen.okButton.classList.add('disable');
+      Awesomescreen.renameConfirmButton.classList.add('disable');
+    }
+  },
+
+  isInputAreaValid: function awesomescreen_isInputAreaValid() {
+    return (Awesomescreen.inputArea.value &&
```

Comment:
We need to check input's validity in javascript to update other dom element's class.